src/test/java/com/epam/tc/hw2/Utils.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.tc.hw2;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Utils {
+
+    public static final String PAGE_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    public static final String HOME_PAGE_TITLE = "Home Page";
+    public static final String LOGIN = "Roman";
+    public static final String PASSWORD = "Jdi1234";
+    public static final String USERNAME = "ROMAN IOVLEV";
+    public static final List<String> headerItems = List.of(
+            "HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"
+    );
+    public static final int IMAGES_NUMBER = 4;
+    public static final List<String> iconsText = List.of(
```

Comment:
All final constants should be in upper case

Reply:
Done

Reply:
Should I create Data classes as Data Providers?